feat(content): Pirate Expansion - Remains of the CCOR (#9597)
endless-sky/endless-sky: data/human/weapons.txt

	description "	The weapon's high rate of fire also makes it useful for saturating well-protected ships' anti-missile defenses so that more dangerous projectiles fired by allied ships can sneak through."
	description "	Mini pods enable fighters and other light craft to carry small quantities of Javelins into battle while retaining enough space for other weapons. Large ships are better off with full-size pods."

outfit "Javelin Turret"
	category "Secondary Weapons"
	cost 110000
	thumbnail "outfit/javelin turret"

> 	"mass" 31

Reviewer: ```suggestion
	"mass" 15
```
As 400 javelins have a total of 16 mass, this way the turret's mass matches its outfit space when fully loaded. That's the standard for missile weapons.

	description "	The weapon's high rate of fire also makes it useful for saturating well-protected ships' anti-missile defenses so that more dangerous projectiles fired by allied ships can sneak through."
	description "	Mini pods enable fighters and other light craft to carry small quantities of Javelins into battle while retaining enough space for other weapons. Large ships are better off with full-size pods."

outfit "Javelin Turret"
	category "Secondary Weapons"
	cost 110000
	thumbnail "outfit/javelin turret"
	"mass" 15
	"outfit space" -31
	"weapon capacity" -31
	"turret mounts" -1
	"javelin capacity" 400
	weapon
		sprite "projectile/javelin"
		"hardpoint sprite" "hardpoint/javelin turret"
		"hardpoint offset" 9.
		sound "javelin"
		ammo "Javelin"

> 		icon "icon/javelin"

Reviewer 2: I think it would make sense to have a different icon for the turret, since having both the gun and the turret installed at the same time will have indistinguishable icons.

		"hit effect" "tiny explosion"
		"inaccuracy" 4
		"turret turn" 2
		"velocity" 10
		"lifetime" 60

> 		"reload" 7

Reviewer 2: I believe that 7.5 would function just fine and result in cleaner numbers in the UI. I understand that the turret sprite has 2 javelin pods and then some worth of barrels; I think it's fine to keep the sprite as-is is if the reload changes.

		"firing energy" .2
		"firing heat" 12
		"shield damage" 48
		"hull damage" 26
		"hit force" 50
		"missile strength" 3

> 	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as other weapons that don't require the use of ammunition to fall back on once depleted of Javelins."

Reviewer 3: ```suggestion
	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."
```

> 	description "	As with its regular pod versions, the Javelin turret is excellent at overwhelming anti-missile systems, especially due to having just over double the fire rate, so that other ordnance from allied ships is able to make it through to the target."

Reviewer 3: ```suggestion
	description "	As with the pod version, the Javelin turret excels at overwhelming anti-missile systems - especially due to its high fire rate, which is just over double that of the Javelin pod - so that other ordnance from allied ships is able to reach the target."
```

Author: The dash seems odd to be me here.

Author: I wasn't sure if it was correct usage, but it appears it is, so seems fine.

> 	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."
> 	description "	As with the pod version, the Javelin turret excels at overwhelming anti-missile systems - especially due to its high fire rate, which is just over double that of the Javelin pod - so that other ordnance from allied ships is able to reach the target."

Reviewer 4: ```suggestion
	description "Mounting not one but two Javelin Pods, the Javelin Turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin Storage Crates, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."
	description "	With just over twice the fire rate of a Javelin Pod, the Javelin Turret excels at overwhelming anti-missile systems so that other ordnance from allied ships is able to reach the target."
```
See #8357 for capitalisations.

The second paragraph read very weirdly to me, and I've juggled it around a bit, but I'd like to get a second opinion on this one.

Reviewer 3: You're right. This is better and more readable than the text with the dashes.

> 	description "	As with its regular pod versions, the Javelin turret is excellent at overwhelming anti-missile systems, especially due to firing twice the speed, so that other ordinance from allied ships is able to make it through to the target."

Reviewer: ```suggestion
	description "	As with its regular pod versions, the Javelin turret is excellent at overwhelming anti-missile systems, especially due to firing at just over twice the speed, so that other ordinance from allied ships is able to make it through to the target."
```

> 	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down and endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expensive of needing to refill on ammo once depleted due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as other weapons that don't require the use of ammunition, so that the parent ship doesn't need to leave combat once depleted of Javelins."
> 	description "	As with its regular pod versions, the Javelin turret is excellent at overwhelming anti-missile systems, especially due to firing at just over twice the speed, so that other ordinance from allied ships is able to make it through to the target."

Reviewer 2: ```suggestion
	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as other weapons that don't require the use of ammunition to fall back on once depleted of Javelins."
	description "	As with its regular pod versions, the Javelin turret is excellent at overwhelming anti-missile systems, especially due to having just over double the fire rate, so that other ordnance from allied ships is able to make it through to the target."
```

	description "	The weapon's high rate of fire also makes it useful for saturating well-protected ships' anti-missile defenses so that more dangerous projectiles fired by allied ships can sneak through."
	description "	Mini pods enable fighters and other light craft to carry small quantities of Javelins into battle while retaining enough space for other weapons. Large ships are better off with full-size pods."

outfit "Javelin Turret"
	category "Secondary Weapons"
	cost 110000
	thumbnail "outfit/javelin turret"
	"mass" 15
	"outfit space" -31
	"weapon capacity" -31
	"turret mounts" -1
	"javelin capacity" 400
	weapon
		sprite "projectile/javelin"
		"hardpoint sprite" "hardpoint/javelin turret"
		"hardpoint offset" 9.
		sound "javelin"
		ammo "Javelin"
		icon "icon/javelin turret"
		"hit effect" "tiny explosion"
		"inaccuracy" 4
		"turret turn" 2
		"velocity" 10
		"lifetime" 60
		"reload" 7.5
		"firing energy" .2
		"firing heat" 12
		"shield damage" 48
		"hull damage" 26
		"hit force" 50
		"missile strength" 3

> 	description "Mounting not one but two Javelin Pods, the Javelin Turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin Storage Crates, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."

Reviewer 2: 50 seconds of fire isn't exactly an endless stream.
```suggestion
	description "Mounting not one but two Javelin Pods, the Javelin Turret delivers a torrential downpour of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin Storage Crates and weapons that don't require the use of ammunition to fall back on once all of the missiles are expended."
```

> 	description "	With twice the fire rate of a Javelin Pod, the Javelin Turret excels at overwhelming anti-missile systems so that other ordnance from allied ships is able to reach the target."

Reviewer 2: I'm not really sure why we have this same blurb in all three Javelin weapon descriptions. Is this really such a standout feature that every single Javelin weapon needs to mention it? I'd say either we leave this, or remove it from both the turret and the mini pod.

Author: I'd be fine if we just removed it.

Reviewer 4: ```suggestion
```
If you're so inclined.

		"hit effect" "bullet impact"
	description `When Dr. Richard J. Gatling invented the rapid-fire gun bearing his name in the 19th century, he hoped that its destructiveness would lead to the end of war itself. Over a millennium later, this modern incarnation of the Gatling Gun sees action in the stars, a testament to its fundamentally robust design - and Dr. Gatling's naivete. While capable of a high fire rate, this gun needs to cool down for twice as long as it fires, and can only sustain a maximum of three seconds of continuous fire.`

outfit "Gatling Turret"
	category "Secondary Weapons"
	cost 120000
	thumbnail "outfit/gat turret"
	"mass" 11
	"outfit space" -23
	"weapon capacity" -23
	"turret mounts" -1
	"gatling round capacity" 6000
	weapon
		sound "gatling"
		"hardpoint sprite" "hardpoint/gat turret hardpoint"
		"submunition" "Gat Submunition"
			"offset" 1.5 -6
			"facing" 0.3
		"submunition" "Gat Submunition"
			"offset" -1.5 -6
			"facing" -0.3
		ammo "Gatling Gun Ammo" 2

> 		icon "icon/gat"

Reviewer 2: Ditto. Something as simple as having three projectiles stacked together in the icon instead of just one would work.

		"inaccuracy" 2
		"turret turn" 3.4
		"velocity override" 16

> 		"lifetime" 0

Reviewer 2: The lowest the lifetime can actually go is 1.

Author: I was mainly copying off of what the Twin Blaster does, so I'll change that to 1 as well.

		"reload" 3
		"burst count" 180
		"burst reload" 1
		"firing energy" .06
		"firing heat" 5
		"cluster"
		"hit effect" "bullet impact"

> 	description `In an act that would make Dr. Richard J. Gatling spin with the ferocity of the barrels of his invention in the grave, the Gatling Turret was born when an enterprising pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Gun has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`

Reviewer 4: ```suggestion
	description `In an act that would make Dr. Richard J. Gatling spin with the ferocity of the barrels of his invention in the grave, the Gatling Turret was born when a resourceful pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Turret has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`
```
Now that you mention it, enterprising is already used in the Twin Modified Blaster's description and we should avoid the repetition here, and this description should really talk about the Gatling Turret's proliferation, if the Gatling Gun was already popular.

~~Yup, reviewing my own description.~~

> 	description `In an act that would make Dr. Richard J. Gatling spin with the ferocity of the barrels of his invention in the grave, the Gatling Turret was born when a resourceful pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Turret has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`

Reviewer 3: ```suggestion
	description `In an act that would make Dr. Richard J. Gatling spin in his grave with the ferocity of the barrels of his invention, the Gatling Turret was born when a resourceful pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Turret has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`
```
Easier to read imo

outfit "Gat Submunition"
	weapon
		sprite "projectile/tinyflare"
		"hit effect" "bullet impact"
		"velocity" 16
		"lifetime" 1

> 		"submunition" "gbullet"

Reviewer 2: This adds an extra frame of delay as to when the visible projectiles are created by having a one-frame, zero velocity projectile be what spawns these submunitions that have the actual velocity, which can throw off the aim of the turret. Since the purpose of this is to have two tinyflare sprites appear, perhaps instead create a double tinyflare sprite that perfectly aligns with the barrels of the hardpoint sprite. The submunitions of the turret can then just be gbullet.

		"firing heat" 5
		"cluster"
		"hit effect" "bullet impact"

> 	description `In an act that would make Dr. Richard J. Gatling spin with the ferocity of the barrels of his invention in the grave, the Gatling Turret was born when an enterprising pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Gun has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`

Reviewer 3: I think this first sentence is perhaps a bit too evocative lol

Author: It makes perfect sense for a Pirate weapon, similar to some other silly descriptions in the game.
